src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/FloodlightKafkaConnectionSpec.groovy

```diff
@@ -121,7 +123,8 @@ class FloodlightKafkaConnectionSpec extends HealthCheckSpecification {
             assert islUtils.getIslInfo(isls, isl.reversed).get().state == IslChangeType.FAILED
         }
 
-        and: "Cleanup: restore the broken link"
+        cleanup:
+        regionsOut && regions.each { lockKeeper.reviveFloodlight(it) }
```

Comment:
how do we know that `floodlight` is available again? probably we should add extra check like `assert northbound.activeSwitches.size() == topology.activeSwitches.size() `

Reply:
done